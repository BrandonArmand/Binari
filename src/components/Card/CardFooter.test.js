import React from "react";
import { render, screen } from "@testing-library/react";
import "@testing-library/jest-dom";
import CardFooter from "./CardFooter";

describe("CardFooter", () => {
    it("renders snapshot", () => {
        const { container } = render(<CardFooter />);
        expect(container).toMatchSnapshot();
    });
    it("renders CardFooter", () => {
        render(<CardFooter>CardFooter</CardFooter>);
        expect(screen.getByText("CardFooter")).toBeInTheDocument();
    });
});
