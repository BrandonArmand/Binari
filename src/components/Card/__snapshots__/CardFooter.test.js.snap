// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardFooter renders snapshot 1`] = `
<div>
  <div
    class="makeStyles-cardFooter-1"
  />
</div>
`;
